examples/tutorial_derive/04_01_enum.md

```diff
@@ -5,7 +5,7 @@ A simple to use, efficient, and full-featured Command Line Argument Parser
 Usage: 04_01_enum_derive[EXE] <MODE>
 
 Arguments:
-  <MODE>  What mode to run the program in [possible values: fast, slow]
+  <MODE>  What mode to run the program in [possible values: fast (alias: quick), slow]
```

Comment:
Feel like this is too many layers of parenthetical statements.

We should either flatten them or reserve them for long help.

I lean towards flattening.

Reply:
@epage I think it would be confusing to put all the aliases into a single flat list, without indicating what value they're aliases for.

No objections to only showing them in long help, though.

Reply:
As this is more being driven by your needs, I'll defer to your preference until we get wider feedback

Reply:
@epage FWIW, while I have a need for visible aliases, I have *no* particular preferences regarding how to display them, as long as they're displayed and it's clear what they're an alias for. I do think the brief help may just be too short to reasonably show them.